code/modules/religion/rites.dm

```diff
@@ -109,6 +109,30 @@
 	human2borg.visible_message("<span class='notice'>[human2borg] has been converted by the rite of [name]!</span>")
 	return TRUE
 
+/datum/religion_rites/upgrade_blessing
+	name = "Receive Blessing"
+	desc = "Receive a blessing from the machine gods to further your ascension"
+	ritual_length = 15 SECONDS
+	ritual_invocations = list("With hammer and faith our bodies were forged",
+						"Oh, Machine-God, We are your instruments of faith",
+						"Let your energy overload our iron-forged blows")
+	invoke_msg = "The end of flesh is near!"
+	favor_cost = 2500
+
+datum/religion_rites/upgrade_blessing/invoke_effect(mob/living/user, atom/movable/religious_tool)
+	..()
+	var/altar_turf = get_turf(religious_tool)
+	var/blessings = pick(/obj/item/organ/cyberimp/arm/surgery,
```

Comment:
you might want to add the (engineering) toolset implant to the pool as well, especially since the robotic organ manipulation surgery steps heavily favor engineering tools over standard surgery ones.

Reply:
i didnt want to add the engineering tools because then it might get too powergamey i want it to be a way to mostly help others